package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "description": "Various examples of working with typescript-eslint. 📝",
   "main": "index.js",
-  "repository": "https://github.com/typescript-eslint/typescript-eslint-examples",
+  "repository": "typescript-eslint/typescript-eslint-examples",
```

Comment:
Why remove the URL base?

Reply:
Heh I only recently learned this was a thing. https://github.com/JoshuaKGoldberg/eslint-plugin-package-json/issues/71